src/skeletal_model.c

```diff
@@ -45,7 +45,11 @@ void SkeletalModel_Clear(skeletal_model_p model)
 void SkeletalModel_GenParentsIndexes(skeletal_model_p model)
 {
     int stack = 0;
+#ifdef _WIN32
```

Comment:
What about Win64? I implemented this patch too but it seems TeslaRus rejected it.

Reply:
working on windows x64 so i dont know because TeslaRus dont view it for now.